SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/Digital/PPMU.cs

```diff
@@ -120,6 +149,21 @@ public static void ForceVoltage(this DigitalSessionsBundle sessionsBundle, IDict
             });
         }
 
+        /// <summary>
+        /// Forces voltage on the target pin(s).
```

Comment:
Suggest changing to: "Forces voltage using specified pin- and site-unique source settings."